sklearn/utils/extmath.py

```diff
@@ -664,7 +664,7 @@ def _incremental_mean_and_var(X, last_mean=.0, last_variance=None,
 
     last_variance : array-like, shape: (n_features,)
 
-    last_sample_count : int
+    last_sample_count : array-like, shape (n_features,)
```

Comment:
The default is still 0, which will actually never work. 
So I would simply remove all defaults and make it all positional keywords (they are always used like that, so that shouldn't change anything for the rest)